fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/common/savings/SavingsAccountHelper.java

```diff
@@ -269,6 +269,12 @@ public Integer addChargesForSavings(final Integer savingsId, final Integer charg
                 getPeriodChargeRequestJSON(chargeId, addDueDate, amount), CommonConstants.RESPONSE_RESOURCE_ID);
     }
 
+    public Integer addChargesForSavingsWithDueDate(final Integer savingsId, final Integer chargeId, String addDueDate,Integer amount) {
+        System.out.println("--------------------------------- ADD CHARGES FOR SAVINGS --------------------------------");
```

Comment:
remove `System.out.println` (just completely remove it if it's a left over you used in testing but don't really need; or replace it with a debug log if you think it could ever be useful to anyone in production)

Reply:
Resolving as this is done.   @andrew-dzak FYI in the future you can (should) just _Resolve conversation_ yourself whenever you amend PRs and have taken review feedback into account - this makes it easier for reviewers.